### ISSUE TITLE
obi go [remote] fails when only ed25519 key has been deployed

### ISSUE THREAD

**Issue by Gobiner** [Oblong/obi#58]
```
└[~/sprockets]> obi go hex --dry-run
ssh -t oblong@10.137.124.207 "mkdir -p /tmp/arice/sprockets"
ssh -t oblong@10.137.124.206 "mkdir -p /tmp/arice/sprockets"
ssh -t oblong@10.137.124.205 "mkdir -p /tmp/arice/sprockets"
ssh -t oblong@10.137.124.204 "mkdir -p /tmp/arice/sprockets"
ssh -t oblong@10.137.124.203 "mkdir -p /tmp/arice/sprockets"

Fatal error: Needed to prompt for a connection or sudo password (host: 10.137.124.204), but input would be ambiguous in parallel mode

Aborting.

Fatal error: Needed to prompt for a connection or sudo password (host: 10.137.124.203), but input would be ambiguous in parallel mode

Aborting.

Fatal error: Needed to prompt for a connection or sudo password (host: 10.137.124.207), but input would be ambiguous in parallel mode

Aborting.

Fatal error: Needed to prompt for a connection or sudo password (host: 10.137.124.206), but input would be ambiguous in parallel mode

Aborting.

Fatal error: Needed to prompt for a connection or sudo password (host: 10.137.124.205), but input would be ambiguous in parallel mode

Aborting.

Fatal error: One or more hosts failed while executing task 'build_task'

Aborting.
```

**Comment by tomjakubowski** [comment]
I think this is a wontfix, probably a paramiko bug?

**Comment by sandover** [comment]
Your call.

**Comment by kyletmiller** [comment]
I just ran into this. Probably a low priority for y'all, but thought I'd +1 it.

Generating an RSA key now just for obi...
